allauth/socialaccount/providers/quickbooks/provider.py

```diff
@@ -12,6 +12,10 @@
 class QuickBooksAccount(ProviderAccount):
     def to_str(self):
         dflt = super(QuickBooksAccount, self).to_str()
+        email = self.account.extra_data.get("email")
+        email_verified = self.account.extra_data.get("emailVerified")
```

Comment:
Checking for `emailVerified` is not needed.

Reply:
Done.